test/integration/ggrc/models/mixins/test_with_readonly_access.py

```diff
@@ -297,8 +297,7 @@ class TestWithReadOnlyAccessFolder(TestCase):
 
   FOLDERABLE_MODEL_NAMES = list(m.__name__ for m in all_models.all_models
                                 if issubclass(m, mixins.Folderable) and
-                                m not in (all_models.Directive,
-                                          all_models.SystemOrProcess))
+                                m not in (all_models.Directive))
```

Comment:
This will raise an error since `(all_models.Directive)` is not treated as a `tuple` but as `all_models.Directive` which is not an iterable. This is actually the reason why integration tests are failing now.
```suggestion
                                m not in (all_models.Directive,))
```
or
```suggestion
                                m is not all_models.Directive)
```

Reply:
Updated with first suggestion